src/sentry/static/sentry/app/main.tsx

```diff
@@ -39,16 +39,22 @@ class Main extends React.Component<Props, State> {
     }
   }
 
+  renderCore() {
```

Comment:
@billyvg is this change cool with you?

Reply:
I don't love overloading `Main` like this, I would consider:

 * ThemeProvider + Global Styles = `<AGoodNameForThisComponent>`
 * Main => `<AGoodNameForThisComponent><Router ...><../>`
 * Pipeline => replace Main with AGoodNameForThisComponent